src-docs/src/views/collapsible_nav/collapsible_nav_all.tsx

```diff
@@ -197,7 +204,16 @@ const CollapsibleNavAll = () => {
       <EuiHorizontalRule margin="none" />
 
       {/* BOTTOM */}
-      <EuiFlexItem className="eui-yScroll">
+      <EuiFlexItem
+        className="eui-yScroll"
+        css={css`
+          @media (max-height: 15em) {
+            flex-shrink: 0 !important;
+            flex-grow: 1 !important;
+            flex-basis: auto !important;
+          }
+        `} // Accessibility - Allows nav items to be seen and interacted with on very small screen sizes
+      >
```

Comment:
same commit nit, also some CSS cleanup. We don't need the `!important` after all
```suggestion
      <EuiFlexItem
        className="eui-yScroll"
         // Accessibility - Allows nav items to be seen and interacted with on very small screen sizes
        css={css`
          @media (max-height: 15em) {
            flex: 1 0 auto;
          }
        `}
      >
```